README.md

```diff
@@ -137,17 +141,20 @@ OR
 2.	`Direction` : Open user default map service and show direction from User location to somewhere.
 
 
-## Changelog
+## Repository
```

Comment:
This isn't aligned with the other repositories. There shouldn't be a Repository header.

Reply:
Check this for the guidelines: https://dev.azure.com/nventive/Practice%20committees/_wiki/wikis/Committees.wiki/14813/Recipes?anchor=repository-structure-and-documentation